src/NzbDrone.Core/MediaFiles/EpisodeImport/ImportApprovedEpisodes.cs

```diff
@@ -122,9 +122,34 @@ public List<ImportResult> Import(List<ImportDecision> decisions, bool newDownloa
 
                     _eventAggregator.PublishEvent(new EpisodeImportedEvent(localEpisode, episodeFile, oldFiles, newDownload, downloadClientItem));
                 }
+                catch (DirectoryNotFoundException e)
+                {
+                    _logger.Warn(e, "Couldn't import episode " + localEpisode);
+
+                    var errorMessage = "Failed to import episode";
+                    if (e.Message.Contains("Root folder"))
+                    {
+                        errorMessage += ", Root folder missing.";
+                    }
+
+                    importResults.Add(new ImportResult(importDecision, errorMessage));
+                }
+                catch (IOException e)
+                {
+                    _logger.Warn(e, "Couldn't import episode " + localEpisode);
+
+                    var errorMessage = "Failed to import episode";
+                    if (e.Message.Contains("Destination already exists."))
```

Comment:
Similar to the above, can we throw a better exception here? Perhaps pre-check if the destination exists and throw our own exception. Parsing a message we're not controlling won't work if the language is not English.